### DISCUSSION TITLE
Scan nearby bl devices

### DISCUSSION THREAD

**Discussion by DoplerGanger** [custom-components/ble_monitor#1041] [Q&A] [answered]
Hola!
Can i use thise component as a presence monitor?
Like . 24/7 scan nearby devices, just like phone scan nearby devices when you try to share some pictures wia bluetooth ?
Sory if it always be in documentation. Srsly i tryed to find it myself ^_^

**Comment by Ernst79** [comment]
https://custom-components.github.io/ble_monitor/configuration_params#track_device

Use the track_device parameter to track a device with a Fixed MAC address. Note that a phone generally doesn't have a fixed MAC address, so, it is not possible to track phones.

**Reply by DoplerGanger** [reply]
In thise looong window i use mac adress and check "Track device"
Other parameters like "temperature humidity " and other just ignore, right?
![image](https://user-images.githubusercontent.com/75626861/196384192-feb885e6-4aad-4e0a-b61c-affdf47ca24b.png)
P.S. I reboot some tymes bluetooth in phone and its still same.
P.S.S. If it true can i monitor device name who still static ???

**Comment by Ernst79** [comment, marked as answer]
Yes, you can ignore the other settings, except the timer option, which can be used to set the device to "away" after x seconds. 

You can't monitor devices based on the name, only on the MAC address or UUID (for iBeacon devices)

**Reply by DoplerGanger** [reply]
Ty kind stranger.
